src/components/CookieBanner/style.scss

```diff
@@ -0,0 +1,121 @@
+@import '~style/vars.scss';
+
+$bgColor: #21435a;
+$textColor: #ffffff;
+$acceptButtonColor: #23d160;
```

Comment:
Can you move these variables into vars.scss and maybe try to use existing css vars?

Reply:
@andre-meyer why? It's not global variables, they're used only in this component